AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -315,6 +315,30 @@ val testExamples = task<Test>("testExamples") {
     testLogging {
         events("passed")
     }
+    filter {
+        excludeTestsMatching("software.amazon.cryptography.example.hierarchy.concurrent.*")
+    }
+}
+
+val testConcurrentExamples = task<Test>("testConcurrentExamples") {
+    description = "Runs examples tests."
+    group = "verification"
+
+    testClassesDirs = sourceSets["testExamples"].output.classesDirs
+    classpath = sourceSets["testExamples"].runtimeClasspath + sourceSets["examples"].output + sourceSets.main.get().output
+    // This will show System.out.println statements
+    testLogging.showStandardStreams = true
+    useTestNG() {
+        suites("src/testExamples/java/software/amazon/cryptography/example/hierarchy/concurrent/testng-parallel.xml")
+        maxParallelForks = 2
+    }
+
+    testLogging {
+        events("passed")
+    }
```

Comment:
If you are running 1000 + 150 + more tests, you may not want to log every pass.

Reply:
these concurrency tests are expensive and I don't want them to run every pr - if i have them run once a week, I don't see an issue in logging the passes.

Reply:
The issue is that you wont be able to see the trees from the forest.

If you have hundreds of lines of "test passed",
it is harder to find the "test failed". 

Regardless of how often these tests are run,
over logging is noise.